src/systems/systems/src/pressurization/cabin_pressure_controller.rs

```diff
@@ -409,9 +450,8 @@ impl<S> PressureSchedule<S> {
         self
     }
 
-    fn new<T, U: Fn() -> S>(from: PressureSchedule<T>, ctor_fn: U) -> Self {
+    fn new<T, U: Fn() -> S>(_from: PressureSchedule<T>, ctor_fn: U) -> Self {
```

Comment:
**thought**: The linter told you one option for the unused variable was prefixing it with an underscore, another common option is to simply put an underscore. 
**suggestion**: I usually decide for the version with `_name` when providing an (empty) default trait function implementation, as in those situations I want implementors of said trait to use that name in their implementation. When working with a concrete function, I usually just use `_`, as there's no value in the variable having any name.

```suggestion
    fn new<T, U: Fn() -> S>(_: PressureSchedule<T>, ctor_fn: U) -> Self {
```

Reply:
Changed to `_`.